core-bundle/src/Resources/config/services.yml

```diff
@@ -769,6 +769,12 @@ services:
         calls:
             - [setName, [contao_frontend]]
 
+    contao.simple_tokens.parser:
+        class: Contao\CoreBundle\SimpleTokens\Parser
```

Comment:
I would put it in the `Util` namespace and name the class `SimpleTokenParser`. And then that makes for `contao.util.simple_token_parser`.